crates/sui-framework/src/lib.rs

```diff
@@ -12,135 +12,118 @@ use sui_types::{
     error::SuiResult,
     move_package::MovePackage,
     object::{Object, OBJECT_START_VERSION},
-    MOVE_STDLIB_ADDRESS, SUI_FRAMEWORK_ADDRESS, SUI_SYSTEM_ADDRESS,
+    MOVE_STDLIB_OBJECT_ID, SUI_FRAMEWORK_OBJECT_ID, SUI_SYSTEM_PACKAGE_ID,
 };
 
 pub mod natives;
 
-/// Defines a new system package at `$address` (an ObjectID), and a type with name `$Package` that
-/// implements the `SystemPackage` trait to give access to the package's contents.
-///
-/// The package's modules are expected to be found at sub-directory `$path` of the the cargo output
-/// directory.  The process of getting them there is usually managed by this crate's `build.rs`
-/// script.
-///
-/// The remaining `$Dep` arguments reference the types for other system packages that are transitive
-/// dependencies of this package.
-macro_rules! define_system_package {
-    ($address:expr, $Package:ident, $path:literal, [$($Dep:ident),* $(,)?]) => {
-        pub struct $Package;
-
-        impl SystemPackage for $Package {
-            const ID: ObjectID = ObjectID::from_address($address);
-
-            const BCS_BYTES: &'static [u8] = include_bytes!(concat!(env!("OUT_DIR"), "/", $path));
-
-            fn transitive_dependencies() -> Vec<ObjectID> {
-                vec![$($Dep::ID,)*]
-            }
-
-            fn as_bytes() -> Vec<Vec<u8>> {
-                bcs::from_bytes($Package::BCS_BYTES).unwrap()
-            }
-
-            fn as_modules() -> &'static [CompiledModule] {
-                static MODULES: Lazy<Vec<CompiledModule>> = Lazy::new(|| {
-                    // deserialization here uses overall max version of supported Move bytecode
-                    // rather than the max supported by the framework but it's OK
-                    $Package::as_bytes().into_iter().map(|m| CompiledModule::deserialize(&m).unwrap()).collect()
-                });
-                &Lazy::force(&MODULES)
-            }
-
-            fn as_package() -> MovePackage {
-                MovePackage::new_system(
-                    OBJECT_START_VERSION,
-                    &$Package::as_modules(),
-                    $Package::transitive_dependencies(),
-                )
-            }
-
-            fn as_object() -> Object {
-                Object::new_system_package(
-                    $Package::as_modules(),
-                    OBJECT_START_VERSION,
-                    $Package::transitive_dependencies(),
-                    TransactionDigest::genesis(),
-                )
-            }
-        }
-    };
-}
-
-define_system_package!(MOVE_STDLIB_ADDRESS, MoveStdlib, "move-stdlib", []);
-define_system_package!(MOVE_STDLIB_ADDRESS, MoveStdlibTest, "move-stdlib-test", []);
-
-define_system_package!(
-    SUI_FRAMEWORK_ADDRESS,
-    SuiFramework,
-    "sui-framework",
-    [MoveStdlib]
-);
-
-define_system_package!(
-    SUI_FRAMEWORK_ADDRESS,
-    SuiFrameworkTest,
-    "sui-framework-test",
```

Comment:
I am slightly surprised that deleting these test ones here didn't cause any issue. I guess these were never used or needed in the first place?